script.js

```diff
@@ -0,0 +1,19 @@
+const hamburger = document.querySelector('.hamburger');
```

Comment:
- [ ] nice work. please kindly use the correct background color `(#6070FF)` for the menu popup so it can follow the figma design

Reply:
- [ ] This issue is not yet resolved :x:

Reply:
- Please, kindly use the correct color for the mobile menu as requested by the previous reviewer.

Reply:
**[OPTIONAL]**

- There don't seem to be any of the nav links  in the desktop version except for the logo. Please, kindly fix this issue.

![Screenshot from 2022-12-21 07-10-42](https://user-images.githubusercontent.com/59397159/208834597-8c2ce206-ea51-4f73-8040-7f431c7c4faa.png)